.ci/deps.sh

```diff
@@ -21,7 +21,7 @@ deps_python_gi="glib2.0-dev gobject-introspection libgirepository1.0-dev python3
 deps_perl="perl libperl-critic-perl"
 deps_infer="m4 opam"
 sudo apt-get -y install $deps $deps_python_gi $deps_python_dbus $deps_perl $deps_infer
-
```

Comment:
this may have happened due to running coala, but it is not a change for this commit.